LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs

```diff
@@ -182,14 +182,15 @@ public void SplitTenderExample_LevelUp_Then_Cash_With_Exemptions()
             // POS applies $1.00 pretax discount to check and updates subtotal and tax
             check.ApplyDiscount(100);
 
-            // Create completed order: expected values
-            // tax amount unchanged
-            // spend amount unchanged
-            // exemption amount unchanged
+            // $8.00 (Subtotal)
+            // $0.80 (Tax)
+            // $8.80 (Total)
+            // $9.80 (Total + Applied Discount)
+
             AdjustedCheckValues expectedCompletedOrderValues =
                 new AdjustedCheckValues(
                     spendAmount: 900,
-                    taxAmount: 80,
+                    taxAmount: 0,
```

Comment:
Just checking: the original implementation (before this PR) would result in the expect taxAmount to be something greater than 0 here, correct?